request/util.js

```diff
@@ -121,7 +122,7 @@ define([
 			query = options.query;
 
 		if(data && !skipData){
-			if(typeof data === 'object' && !(data instanceof ArrayBuffer || data instanceof Blob )){
+			if(typeof data === 'object' && (!(has('native-xhr2')) || !(data instanceof ArrayBuffer || data instanceof Blob ))){
```

Comment:
FWIW, I guess the other way would be:

```
if(typeof data === 'object' && (typeof ArrayBuffer === 'undefined' || !(data instanceof ArrayBuffer)) 
&& (typeof Blob === 'undefined' || !(data instanceof Blob )))
```
I just don't know if `ArrayBuffer` and `Blob` are defined iff `has('native-xhr2')` is true.